src/Http/Routing/perf/Matching/MatcherAzureBenchmarkBase.generated.cs

```diff
@@ -476,7 +476,7 @@ private protected void SetupEndpoints()
             Endpoints[459] = CreateEndpoint("/providers/Microsoft.ADHybridHealthService/services/{serviceName}/metricmetadata/{metricName}", "GET");
             Endpoints[460] = CreateEndpoint("/providers/Microsoft.ADHybridHealthService/services/{serviceName}/servicemembers/{serviceMemberId}", "DELETE");
             Endpoints[461] = CreateEndpoint("/providers/Microsoft.ADHybridHealthService/services/{serviceName}/servicemembers/{serviceMemberId}", "GET");
-            Endpoints[462] = CreateEndpoint("/providers/Microsoft.ADHybridHealthService/services/{serviceName}/TenantWhitelisting/{featureName}", "GET");
```

Comment:
This file claims it was generated, but from where?

Reply:
From something that Ryan Nowak did a million years ago.